java/src/jmri/jmrit/vsdecoder/VSDecoder.java

```diff
@@ -451,7 +385,7 @@ public void setPosition(PhysicalLocation p) {
         if (create_xy_series) {   
             log.info("{}: {}\t{}", this.getAddress(), (float) Math.round(p.x*10000)/10000, p.y);
         }
-        log.debug("( " + this.getAddress() + ") Set Position: " + p.toString());
+        log.debug("( " + this.getAddress() + ") Set Position: " + p);
```

Comment:
Since String creation is relatively expensive, can this be rewritten as `log.debug("( {}. ) Set Position: ", this.getAddress(), p);` to avoid the call to `this.getAddress()` and the implicit call to `p.toString()` unless debugging? (In general that is the form all `log.[level](...);` statements should take.)

Reply:
@rhwood  Thank you. I will update the PR shortly.